recipes/wasmtime/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class WasmtimeTestConan(ConanFile):
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    generators = 'cmake', 'cmake_find_package'
```

Comment:
```suggestion
    generators = 'cmake'
```

Reply:
What is the point ? In other package I used it without any issues

Reply:
This is needed for https://github.com/conan-io/conan-center-index/pull/6083#discussion_r670904461

Reply:
But what if I want to use generator as a client of this library ?
I do not what parts of my libraries imported as `cmake_find_package` and other part as `cmake` ...
It would nightmare ...

Reply:
I don't see a problem, if `cmake_find_package` works, test package may use it.
there is no goal to test all the generators. I am personally okay if it uses `pkg_config` or whatever.

Reply:
The idea of test packages of recipes in conan-center-index is to test the imported targets that the upstream project provides.
Because the upstream project does not provide any cmake scripts, no cmake find module should be used/tested.

We're testing the recipe here, not conan features/generators.

Reply:
But what if I want to test using this package by specific generator ...
For me it is subjective changes

Reply:
Conan does that.

We only test what the recipe defines (to ensure upstream is respected)

Reply:
Done, just because I tired of arguing on these subjective changes